Source/Core/VideoBackends/Vulkan/CommandBufferManager.cpp

```diff
@@ -303,7 +303,7 @@ void CommandBufferManager::SubmitCommandBuffer(bool submit_on_worker_thread,
     if (res != VK_SUCCESS)
     {
       LOG_VULKAN_ERROR(res, "vkEndCommandBuffer failed: ");
-      PanicAlertFmt("Failed to end command buffer");
+      PanicAlertFmt("Failed to end command buffer: {} ({})", VkResultToString(res), res);
```

Comment:
`LOG_VULKAN_ERROR` uses `static_cast<int>(res)`, but that probably isn't necessary.

I was going to suggest that it might be better to print it as `{:08x}`, but it looks like [the docs](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkResult.html) show them in decimal. So this is probably good as-is.